src/jit/gentree.h

```diff
@@ -3284,6 +3284,9 @@ struct GenTreeCall final : public GenTree
 #define GTF_CALL_M_TAILCALL_VIA_HELPER 0x0200 // GT_CALL -- call is a tail call dispatched via tail call JIT helper.
 #endif                                        // !LEGACY_BACKEND
 
+#define GTF_CALL_M_FAT_POINTER_CHECK 0x0200 // GT_CALL -- it is calli that needs to be tranformed
```

Comment:
>#define GTF_CALL_M_FAT_POINTER_CHECK 0x0200 [](start = 0, length = 43)

As you can see in this file, for every flag we have routines that check whether that flag is set or not.

e.g. IsTailCallViaHelper()

I see in other source files we are referencing GTF_CALL_M_FAT_POINTER_CHECK instead of using routines that abstract flag check.